recipe/meta.yaml

```diff
@@ -1,42 +1,48 @@
 {% set name = "fast-pt" %}
-{% set version = "3.1.1" %}
+{% set version = "3.2" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://github.com/JoeMcEwen/FAST-PT/archive/v{{ version }}.tar.gz
-  sha256: 6b444fe05817b0f5d2d252f0868bf54f39661f41ca967a6609f1be0e995666ae
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/fast_pt-{{ version }}.tar.gz
+  sha256: ec4cfdb4a352b300ca7ec2f313a73c5676ec61ef665345b0fc37a258f5ecbd5d
 
 build:
   noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   number: 0
-  script: "{{ PYTHON }} -m pip install . -vv"
 
 requirements:
   host:
-    - python >=3.7
+    - python >=3.5
+    - setuptools
+    - wheel
     - pip
   run:
-    - python >=3.7
-    - numpy
-    - scipy
-    - matplotlib-base
+    - python >=3.5
+    - numpy >=1.17
+    - scipy >=1.2
+    - matplotlib-base >=3.0
 
 test:
   imports:
     - fastpt
+  commands:
+    - pip check
+  requires:
+    - pip
 
 about:
-  home: https://github.com/JoeMcEwen/FAST-PT
+  home: https://pypi.org/project/fast-pt/
+  summary: FAST-PT is a code to calculate quantities in cosmological perturbation theory at 1-loop (including, e.g., corrections to the matter power spectrum).
+  dev_url: https://github.com/jablazek/FAST-PT
   license: MIT
-  license_family: MIT
   license_file: LICENSE
-  summary: |
-    FAST-PT is a code to calculate quantities in cosmological perturbation theory
-    at 1-loop (including, e.g., corrections to the matter power spectrum).
 
 extra:
   recipe-maintainers:
     - beckermr
+    - jablazek
```

Comment:
@jablazek must confirm he wants to be a maintainer here.

Reply:
I confirm - I would like to be a maintainer. (Apologies for the delay. I thought I had confirmed already, but I think it was on a different repo). Or did you mean confirm the overall PR?

Reply:
nope this is great - thank you!